operator/v1/types_dns.go

```diff
@@ -287,6 +287,17 @@ type ForwardPlugin struct {
 	//
 	// +optional
 	TransportConfig DNSTransportConfig `json:"transportConfig,omitempty"`
+
+	// protocolPolicy configures the protocol policy for upstream DNS requests.
```

Comment:
Since we already have Policy, maybe just name it Protocol, with options Default and TCP?  And UDP if that is an option not represented by Default.

Reply:
I would be fine with either `protocolPolicy` or `protocolStrategy`.

Reply:
Updated to `protocolStrategy`